packages/react/src/Banner/Banner.examples.stories.tsx

```diff
@@ -46,49 +43,6 @@ export const WithUserAction = () => {
   )
 }
 
-export const WithDynamicContent = () => {
```

Comment:
Moving this to `features` so it gets surfaced under the Examples section on the main docs.

Brought up a potentially preferable alternative on [ slack thread](https://github.slack.com/archives/C01L618AEP9/p1736967795324489)